auth-server/src/controllers/eventsHook.ts

```diff
@@ -121,21 +122,85 @@ const sendProposalCreatedMail = async (onchainLink): Promise<MessageType> => {
 		link = 'referendum';
 	}
 
+	if (onchain_treasury_proposal_id === 0 || onchain_treasury_proposal_id) {
```

Comment:
I guess this should be before motion, because it would override it although a motion could come after a treasury.

Reply:
done